### DISCUSSION TITLE
使用fake-ip后traefik反代失效

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [vernesong/OpenClash#2039] [Q&A]
我目前是在truenas scale上使用docker搭建了cloudreve网盘
配合traefik反代使用
启用openclash的fake-ip后反代失效
redir-host正常
请问这可能是由于什么原因造成的
以及可能的解决方案

**Comment by vernesong** [comment]
设置fakeip-filter

**Reply by shoucandanghehe** [reply]
不懂就问 在哪设置
自定义规则似乎不行（
![image](https://user-images.githubusercontent.com/51957264/153775711-dfc6d87c-3584-4382-97eb-0dff4dd0b506.png)
